### ISSUE TITLE
can not compiled successed.

### ISSUE THREAD

**Issue by MirrorYuChen** [IBM/tfjs-web-app#6]
I have follow the tutorials step by step,when I run 
>> yarn start-dev
some errors accurred:
![image](https://user-images.githubusercontent.com/49817428/69782292-871c7f80-11eb-11ea-9cbb-09b1e99b4964.png)
Can anybody give me some advice to solve it?

**Comment by pvaneck** [comment]
What model did you convert in the "Download and convert pre-trained model" step? I found that certain `h5` model files don't properly convert when I run it through `tensorflowjs_converter`. I ran into the issue you were hitting when I manually downloaded a mobilenet model (https://github.com/fchollet/deep-learning-models/releases/download/v0.6/mobilenet_1_0_224_tf.h5). After converting this model and trying to use it, I would hit the `Cannot read property 'model_config' of null` error you are hitting.

However, when downloading through the model through the Keras API in the `download_model.py` script located this repository, seems that the `h5` file generated through that would convert correctly and can be used. This might have to do with some differences in the `h5` formatting for older saved models.
